values/values.go

```diff
@@ -148,6 +148,52 @@ var (
 	Null = null{}
 )
 
+// Extract the primitive value from the Value interface.
+// This will panic when called on some Values that cannot be unwrapped.
```

Comment:
I don't see the panic in the code implementation. Can you also specify in the comment which values cannot be unwrapped for the documentation?